packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md

```diff
@@ -181,6 +181,7 @@ This endpoint requires an object with the following fields.
 | `require_preauth_for_creds`   | boolean | (Optional) Whether to require preauthorization before an ASA User can retrieve credentials to sign in. Default is `false`. |
 | `shared_admin_user_name`   | string | (Optional) The name for a shared admin user on Servers in this Project. If `force_shared_ssh_users` is `true`, this must be provided. |
 | `shared_standard_user_name`   | string | (Optional) The name for a shared standard user on Servers in this Project. If `force_shared_ssh_users` is `true`, this must be provided. |
+| `ssh_certificate_type`   | string | (Optional) The type of signature algorithm used for authentication keys. Possible values are `CERT_TYPE_ED25519_01` and `CERT_TYPE_RSA_01`. Default is `CERT_TYPE_ED25519_01`. |
```

Comment:
I've been using this construction in the api reference content to present supported values.
```suggestion
| `ssh_certificate_type`   | string | (Optional) The type of signature algorithm used for authentication keys. Supported values: `CERT_TYPE_ED25519_01` and `CERT_TYPE_RSA_01`. Default is `CERT_TYPE_ED25519_01`. |
```